src/h5web/visualizations/shared/models.ts

```diff
@@ -13,6 +13,12 @@ export enum ScaleType {
   SymLog = 'SymLog',
 }
 
+export const SCALE_FUNCTIONS: Record<ScaleType, AxisScaleFn> = {
```

Comment:
To be noted that in `HeatmapVis`, we use the type `DataScaleFn` that has now the same signature as `AxisScaleFn` used in `LineVis`. 
TypeScript does not complain but I propose we merge the two in `ScaleFn` or something similar.